tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -1706,24 +1730,43 @@ object Symbols {
       extends Symbol(owner)
       with DeclaringSymbol {
     import PackageSymbol.*
+    import tastyquery.reader.Loaders.PackageLoadingInfo
 
     type DefiningTreeType = Nothing
     type DeclType = Symbol
 
     private[Symbols] val specialKind: SpecialKind = computeSpecialKind(name, owner)
 
+    /** Package loading info with raw data from the classpath. */
+    private var optLoadingInfo: Option[PackageLoadingInfo] = None
```

Comment:
just checking why this is safe to be a plain var?

Reply:
im assuming that the flow is that it is the thread that initialised the context is the thread that both creates this object and sets this, but then when is the update of setting this field  "published" to other threads

Reply:
Hum, good question. For the members of other `Symbol`s, they should be published thanks to `myDeclarations.compareAndSet` at their `PackageSymbol`'s level. But for the things initialized during context initialization, we don't have a definitive release barrier. We might need to add one at the end of `Context.initialize` perhaps?

Reply:
could be good

Reply:
Done.